_posts/2019-02-11-lecture-08.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: distill
+title: "Lecture 8: Causal Discovery and Inference"
+description: <insert description here>
+date: <insert date here>
+
+lecturers:
+  - name: Eric Xing
+    url: "https://www.cs.cmu.edu/~epxing/"
+
+authors:
+  - name: Abhilasha Ravichander
+    url: "#"  # optional URL to the author's homepage
+  - name: Aniruddh Shetty
+    url: "#"
+  - name: Tom Yan
+    url: "#"
+  - name: Kin Gutierrez Olivares
+
+editors:
+  - name: Lisa Lee
+    url: "http://leelisa.com"
+
+abstract: >
+---
+Causation implies correlation (or dependence), but correlation does not imply causation. Consider the case of chocolate consumption vs. the number of Nobel Laureates in the country:
+
+<figure>
+  <div class="row">
+    <div class="col two">
+      <img src="{{ '/assets/img/notes/lecture-08/scribe1.jpg' | relative_url }}"  width="600" height="400" />
+    </div>
+  </div>
+  <figcaption>
+    <strong>Chocolate Consumption vs. # of Nobel Laureates</strong>
+  </figcaption>
+</figure>
+
+From this plot, we can see that they are highly correlated. But it is absurd to say that if we want to increase the number of Nobel Laureates in a country, we need to eat more chocolate. This is where the idea of causality comes in. They are correlated, but chocolate consumption is not the cause of Nobel Laureates. The following equations capture this difference mathematically:
+
+$$X$$ and $$Y$$ are dependent if and only if $$ \exists x_1 $$ different than $$ x_2 $$ such that:
+
+$$
+P(Y \mid X=x_{1}) \text{ different than } P(Y \mid X=x_2)
```

Comment:
These equations still do not render when I run jekyll locally:

![screen shot 2019-02-28 at 12 02 00 am](https://user-images.githubusercontent.com/2617555/53542350-19241a00-3aec-11e9-8ab8-1edc5d564390.png)

Please make sure that all equations render properly. If my local instance is wrong and the equations render properly on your computer, let me know.

Reply:
@alshedivat pushed a fix and the equations render now. Thanks Maruan!